Jenkinsfile

```diff
@@ -19,13 +19,11 @@ def prefixlessTag(tag) {
 }
 def zipUploadUrl
 def zipName
-def ascUploadUrl
-def ascName
 
 pipeline {
     agent {
         kubernetes {
-            yaml kubePodTemplate(name: 'full_jnlp.yaml')
+            yaml kubePodTemplate(name: 'gauge.yaml')
```

Comment:
As you suggested, I would prefer that we add the signer to `full_jnlp` because using `gauge` feels a bit wrong here. However, i don't think it's a blocker so we can always merge this and raise a follow-on task to add the signer to the full image and then use that here.

Reply:
done in https://github.com/IBM/cloudant-kafka-connector/pull/161/commits/694a9683e3ebe862afda8366b604ce8d12e3afd4